src/main/java/com/qunite/api/security/FilterChainExceptionHandler.java

```diff
@@ -0,0 +1,31 @@
+package com.qunite.api.security;
+
+import jakarta.servlet.FilterChain;
+import jakarta.servlet.http.HttpServletRequest;
+import jakarta.servlet.http.HttpServletResponse;
+import lombok.NonNull;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Component;
+import org.springframework.web.filter.OncePerRequestFilter;
+import org.springframework.web.servlet.HandlerExceptionResolver;
+
+@Component
+public class FilterChainExceptionHandler extends OncePerRequestFilter {
+  private final HandlerExceptionResolver resolver;
+
+  public FilterChainExceptionHandler(@Qualifier("handlerExceptionResolver")
+                                     HandlerExceptionResolver resolver) {
+    this.resolver = resolver;
+  }
+
+  @Override
+  protected void doFilterInternal(@NonNull HttpServletRequest request,
+                                  @NonNull HttpServletResponse response,
+                                  @NonNull FilterChain filterChain) {
+    try {
+      filterChain.doFilter(request, response);
```

Comment:
creating filter to resolve exceptions like DecodedJWTException? do not u thin it seems to be problem of input to pass incorrect jwt token?

Reply:
Right now this filter relates only to jwt exception, but it can allow us handle any exception that occurs in filters, so i think this is appropriate.